Implement new design

jean opened the pull request:
Still needs some touches ... 

- logo in footer is blue-on-blue
- margins on inside pages (not frontpage) needs fixing

bluenex committed:
try to get fullpage.js working

pektoon committed:
add new html and css code for new theme

pektoon committed:
add new html and css code for new theme

pektoon committed:
add html and css code for footer section

pektoon committed:
add html and css code for footer section

bluenex committed:
use html instead of rst for frontpage, tweak design

bluenex committed:
use html instead of rst for frontpage, tweak design

jean committed:
Upgrade NetlifyCMS

jean committed:
Enable editorial_workflow in NetlifyCMS

bluenex committed:
add news and welcom submission page

jean committed:
Create test-page.th.md

bluenex committed:
done styling for first page

bluenex committed:
done news page style

jean committed:
Sync with `master`

jean committed:
Delete RST frontpage

jean committed:
Remove RST frontpage

jean committed:
Merge branch 'recover-style' of github.com:salween/pyconth-www into recover-style

jean committed:
Fix typo, add img alt attribute

bluenex committed:
done sponsor and footer styling

bluenex committed:
copy style to .en page, not fully translate yet

bluenex committed:
fix header typo

jean committed:
Add post shortcode, comment out sections w/o content

Fix some typo too.

jean committed:
Merge branch 'recover-style' of github.com:salween/pyconth-www into recover-style

jean committed:
Merge branch 'master' of github.com:salween/pyconth-www

jean committed:
Add alt for img, add TODO comment

jean committed:
Sync with Thai

bluenex committed:
add countdown for both languages

jean committed:
Merge pull request #7 from salween/recover-style

implement front page design following the sketch

jean committed:
Merge branch 'master' of github.com:salween/pyconth-www

jean committed:
Merge branch 'master' of github.com:PyConTH/www

jean committed:
Fix links, use shortcodes

Commit:
Formatting

Commit:
Formatting

Commit:
Remove fullpage.js

Commit:
Add palm tree image

Commit:
Make only 1 h1 element per page

Commit:
Add Istok Web default font from Google Fonts

Commit:
Add TailwindCSS

Commit:
Fixes top section to follow spec

Commit:
Fix News section per spec

Commit:
Fix row class negative margins overflowing width

Commit:
Fix button styles

Commit:
Fix submission welcome section and button styles

Commit:
Fix speaker section

Commit:
Fix about section

jean committed:
Merge pull request #8 from Macavirus/frontpage

Fixes Frontpage design to spec

bluenex commented:
@jean `.th` site isn't done yet? Also the footer seems to be stretched to be too longgg (Chrome only, see attached image).

![image](https://user-images.githubusercontent.com/11027706/54258718-0e5e9000-4596-11e9-8cb6-34ad81f84f8a.png)

djay commented:
@bluenex more than the image streched. that footer isn't actually the footer and appearing on all pages. it needs to replace the other footer.

bluenex commented:
@djay yeah it's not in `conf.py` yet. Need to work on it as well.

Commit:
Make TH frontpage match EN frontpage

Commit:
Make TH frontpage match EN frontpage

Commit:
Fix mis-nested tags

Commit:
Fix too-tall naga logo height on Chrome

Commit:
Fix Footer styles

Commit:
Remove Second footer in EN template

Commit:
Remove second footer in TH template

Commit:
Fix button occluding headline

Commit:
Fix max-width on all pages

jean committed:
Merge branch 'master' of github.com:PyConTH/www

jean committed:
Merge branch 'master' of github.com:Macavirus/www

bluenex commented on site/conf.py:
```diff
@@ -997,32 +997,37 @@
-    <div class="col-md text-center">
-        <img src="/logo.png" class="footer-logo" alt="PyCon Thailand" />
+  <div class="row mx-4 my-4">
+    <div class="row">
```
Refer to this [SO thread](https://stackoverflow.com/a/34006377/4010864). This solves the stretching image issue on Chrome. We can also override `.row` which I'm not sure that will affect any other components of the site.

```suggestion
    <div class="row" style="align-items: flex-start;">
```

jean replied:
OK, applied

jean committed:
Solve the issue stretching image issue on Chrome

Refer to this [SO thread](https://stackoverflow.com/a/34006377/4010864).
This solves the issue stretching image issue on Chrome.
We can also override `.row` which I'm not sure that will affect any other components of the site.

```suggestion
    <div class="row" style="align-items: flex-start;">
```

(From @bluenex)

jean committed:
Merge branch 'master' of github.com:PyConTH/www

jean committed:
The footer isn't RST
